postfix.md

```diff
@@ -0,0 +1,58 @@
+# Configuring postfix as a simple SMTP relay
+
+## Installation
+You will need to make sure the following packages are installed:
+* libsasl2-2
+* libsasl2-modules
+* mailutils
+`sudo apt install libsasl2-2 libsasl2-modules mailutils`
```

Comment:
Remove if these are installed automatically. (If they are not, it is a bug in Debian or Ubuntu.)